Codecademy/gamut
# fix(Alert): Update alert to be based on component width

### Overview


Updates alert to be based on component width not screen size, so the button stacks when the alert is xs or smaller


### PR Checklist

- [x] Related to JIRA ticket: [GM-868]

#### Testing Instructions



1. View the Alert story here https://6706b76391cf1c0f2ee9213f--gamut-preview.netlify.app/?path=/docs/molecules-alert--usage-guidelines#small-widths
2. Confirm the alert still looks as expected, but the button stacks when the component width is xs or smaller

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/38632)  | [Monolith Env](https://pr-38632-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/8085)  | [Portal Env](https://tengu.codecademy.com/admin?PR_ENV=admin-pr-8085)   |




[GM-868]: https://skillsoftdev.atlassian.net/browse/GM-868?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
